src/modules/tests/test-slot-attributes/test-slot-attributes.actions.ts

```diff
@@ -10,3 +10,14 @@ export class PopulateTestSlotAttributes implements Action {
 
 export type Types =
   | PopulateTestSlotAttributes;
+
+export const extractTestSlotAttributes = (slotData): TestSlotAttributes => {
```

Comment:
This is not an action. If you wanted to build a selector, please place this in a selector file

Reply:
Thanks. Moved to `test-slot-attributes.selector.ts`.